pdr_backend/lake/gql_data_factory.py

```diff
@@ -285,14 +281,16 @@ def _update(self):
             3. Integrate config/pp if needed
         @arguments
             fin_ut -- a timestamp, in ms, in UTC
+            start_ut_ppss -- a timestamp, in ms, as int
         """
         fin_ut = self.ppss.lake_ss.fin_timestamp
+        start_ut_ppss = self.ppss.lake_ss.st_timestamp
 
         for table in (
             TableRegistry().get_tables(self.record_config["gql_tables"]).values()
         ):
             # calculate start and end timestamps
-            st_ut = self._calc_start_ut(table)
+            st_ut = self._calc_start_ut(table, start_ut_ppss)
```

Comment:
i think this makes sense, but i don't understand how it fixes the problem you described in the ticket

further, you didn't provide any test or other setup showing me how this fixes the problem

Reply:
It fixes the problem by having a constat start timestamp across all the tables while the fetch process is running instead of initiating it at each table level which causes different values.

I have not provided a test because It takes a lot of time and we should validate the approach before implementing the solution e-e and then just ditching it